scripts/test/run.py

```diff
@@ -61,13 +60,3 @@ def restore_src():
         Path(r"_pandas-stubs").rename("pandas-stubs")
     else:
         raise FileNotFoundError("'_pandas-stubs' folder does not exists.")
-
-
-def clean_mypy_cache():
-    if Path(".mypy_cache").exists():
-        shutil.rmtree(".mypy_cache")
-
-
-def clean_pytest_cache():
-    if Path(".mypy_cache").exists():
-        shutil.rmtree(".pytest_cache")
```

Comment:
the test was wrong here, so the `-c` never worked!

Reply:
hahaha :) I didn't notice that when I removed it :)